bin/buildBlueprints.js

```diff
@@ -12,37 +13,36 @@ var makeBuild = require('../lib/makeBuild').makeBuild;
 var configs = require('../lib/configs');
 var getWebpackEntryForTest = require('../lib/getWebpackEntryForTest');
 
+var TEST_DIR = './.test';
```

Comment:
i want to pull this from the actual test configs.  previously this was hardcoded to use the default config, so i figured this was no worse: https://github.com/reddit/node-build/pull/25/files#diff-4e7cbd68bf35721fea2c2cf9ec2385bbL164